shared/torngit/github.py

```diff
@@ -370,12 +370,27 @@ async def list_repos_using_installation(self, username):
                 )
                 if len(res["repositories"]) == 0:
                     break
-                repos.extend(res["repositories"])
+
+                for repo in res["repositories"]:
+                    repos.append(
+                        dict(
+                            owner=dict(
+                                service_id=repo["owner"]["id"],
+                                username=repo["owner"]["login"],
+                            ),
+                            repo=dict(
+                                service_id=repo["id"],
+                                name=repo["name"],
+                                language=self._validate_language(repo["language"]),
+                                private=repo["private"],
+                                branch=repo["default_branch"] or "master",
+                            ),
+                        )
+                    )
+
                 if len(res["repositories"]) < 100:
                     break
 
-            for repo in repos:
-                repo["id"] = str(repo["id"])
```

Comment:
`worker` is currently using this but we can check `repo["repo"]["service_id"]` instead when incorporating this